jjb/paas/paas-jobs.yml

```diff
@@ -97,7 +97,7 @@
     parameters:
       - string:
           name: ORIGIN_VERSION
-          default: '3.6.0'
+          default: '3.7.0'
```

Comment:
I have a small concern here with just using '3.7.0' for the version number, without adding a suffix like 'pre' or 'alpha'